src/app/components/challenge/challenge.component.html

```diff
@@ -1,145 +1,151 @@
+                                <li><a
+                                        [class.selected]="localRouter.url.endsWith('evaluation')"
+                                        [routerLink]="['./evaluation']"
+                                        class="text-light-black fw-semibold fs-15">
+                                        <i class="fa fa-bar-chart-o"></i>
+                                        Evaluation</a></li>
+                                <li><a
+                                        [class.selected]="localRouter.url.endsWith('phases')"
+                                        [routerLink]="['./phases']"
+                                        class="text-light-black fw-semibold fs-15">
+                                        <i class="fa fa-level-up"></i> Phases</a></li>
+                                <li *ngIf="!isParticipated"><a
+                                        [class.selected]="localRouter.url.endsWith('participate')"
+                                        [routerLink]="['./participate']"
+                                        class="text-light-black fw-semibold fs-15">
+                                        <i class="fa fa-free-code-camp"></i>
+                                        Participate</a></li>
+                                <li *ngIf="isParticipated"><a
+                                        [class.selected]="localRouter.url.endsWith('submit')"
+                                        [routerLink]="['./submit']"
+                                        class="text-light-black fw-semibold fs-15">
+                                        <i class="fa fa-upload"></i> Submit</a></li>
+                                <li *ngIf="isParticipated"><a
+                                        [class.selected]="localRouter.url.endsWith('my-submissions')"
+                                        [routerLink]="['./my-submissions']"
+                                        class="text-light-black fw-semibold fs-15">
+                                        <i class="fa fa-eye"></i> My Submissions</a></li>
+                                <li *ngIf="isChallengeHost"><a
+                                        [class.selected]="localRouter.url.endsWith('view-all-submissions')"
+                                        [routerLink]="['./view-all-submissions']"
+                                        class="text-light-black fw-semibold fs-15">
+                                        <i class="fa fa-eye"></i>View All Submissions</a></li>
+                                <li><a
+                                        [class.selected]="localRouter.url.endsWith('leaderboard')"
+                                        [routerLink]="['./leaderboard']"
+                                        class="text-light-black fw-semibold fs-15">
+                                        <i class="fa fa-th-list"></i> Leaderboard</a></li>
+                                <li *ngIf=""><a
```

Comment:
**ngIf** is doing nothing here.

Reply:
oh thanks :) Yeah actually forgot to modify this condition, Updated the condition @lunayach